backend/box/box.go

```diff
@@ -98,6 +128,93 @@ func init() {
 	})
 }
 
+func getBoxConfig(configFile string) ConfigJSON {
+	jsonData, err := os.Open(configFile)
+	if err != nil {
+		log.Fatalf("Failed to read Box config: %v", err)
```

Comment:
Ideally this should return the error upwards.

I'm trying to kill off the log.Fatals in the backends so it would be helpful if you could pass errors up and put the log.Fatal in the Config function where it will be easy to fix when you can return an error from there!